app/build.gradle

```
@@ -102,6 +106,17 @@ dependencies {
 
     testImplementation 'org.robolectric:robolectric:4.3.1'
     testImplementation 'androidx.test:runner:1.3.0'
+
+    // Hilt
+    def hilt_version = '2.31-alpha'
+    implementation "com.google.dagger:hilt-android:$hilt_version"
+    kapt "com.google.dagger:hilt-android-compiler:$hilt_version"
+    def hilt_viewmodel_version = '1.0.0-alpha02'
+    implementation "androidx.hilt:hilt-lifecycle-viewmodel:$hilt_viewmodel_version"
+    kapt "androidx.hilt:hilt-compiler:$hilt_viewmodel_version"
+
+    // Leak Canary
+    debugImplementation 'com.squareup.leakcanary:leakcanary-android:2.6'
```

公式ドキュメントを読む限りだと、 Instrumentaion tests を実行する場合は `androidTestImplementation` も必要そうです。
ref: https://square.github.io/leakcanary/upgrading-to-leakcanary-2.0/

```suggestion
    def leak_canary_version = '2.6'
    debugImplementation "com.squareup.leakcanary:leakcanary-android:$leak_canary_version"
    androidTestImplementation "com.squareup.leakcanary:leakcanary-android-instrumentation:$leak_canary_version"
}
```

---

確かにそうですね。申し訳ないです。